pkg/deploy/defaults.go

```diff
@@ -27,12 +31,10 @@ const (
 	DefaultIngressStrategy              = "multi-host"
 	DefaultIngressClass                 = "nginx"
 	DefaultPluginRegistryImage          = "quay.io/eclipse/che-plugin-registry:7.0.0-RC-2.0"
-	DefaultPluginRegistryPullPolicy     = "Always"
```

Comment:
do you need to also set DefaultXXPullPolicy for Che and identityProvider images?

Reply:
I don't use default values for pull policy, because the default is calculated based on the image tag.

Reply:
ah, of course. I was reading this in reverse (as if you ADDED two PullPolicy values rather than removing them). +1